app/build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 21
+
+    defaultConfig {
+        applicationId "community.fairphone.mycontacts"
+        minSdkVersion 17
+        targetSdkVersion 21
+        versionCode 3
+        versionName "2.1"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+            shrinkResources true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            applicationIdSuffix ".debug"
+        }
+    }
+}
+
+dependencies {
+    api 'com.android.support:support-v4:21.0.3'
+    api 'com.android.support:appcompat-v7:21.0.3'
+    api 'com.googlecode.libphonenumber:libphonenumber:5.5'
```

Comment:
@Roboe why `api`? `implementation` is used for these things.

Reply:
Oh, c'mon, I understood it the other way some days ago (`api` being more restrictive than `implementation`). You are right, I'll drop that commit.